docs/guides/core-concepts/retry-ability.mdx

```diff
@@ -170,15 +169,13 @@ itself failed.
 
 ## Built-in assertions
 
-Often a Cypress command has built-in assertions that will cause the previous
-queries to be retried. For example, the [`.eq()`](/api/commands/eq) query will
-be retried even without any attached assertions until it finds an element with
-the given index.
+Often a Cypress command has built-in assertions that will cause the command to
```

Comment:
```suggestion
Often a Cypress command has [Implicit Assertions](link) that will cause the command to
```

Reply:
I'm not sure it makes sense to link back to the "Getting Started" guide here - this is almost the same content, but written this time with a more granular focus on how commands and queries retry. This is already more detailed than the place it would link.

I have renamed the section "Implicit Assertions" rather than "Built-in Assertions" to better match the terminology in the other guide.